docs/user-guide/menu-bar.md

```diff
@@ -85,7 +85,18 @@ The following example shows the steps required to set up a custom service:
 
     - **Description** is displayed to the bottom of each results row. We use *Manhattan Roads* to point out the source layer name but feel free to change it. You can use the same property placeholder syntax here.
 
-    - **Priority** is a weight assigned to the custom results. It is used to sort all the search results so higher values determine higher positions in the results list. The default [OpenStreetMap Nominatim search engine](https://nominatim.openstreetmap.org/) result has priority equals to 5, so to see our custom results in the first positions we should set a value greater than 5 for this field. We use 6.
+    - **Priority** is a weight assigned to the custom results. It is used to sort all the search results so lower values determine higher positions in the results list (top). The default [OpenStreetMap Nominatim search engine](https://nominatim.openstreetmap.org/) result has priority equals to 5, so to see our custom results in the first positions we should set a value lower than 5 for this field. We use 3.
+
+    - **Launch Info Panel** is a flag that can be used to trigger GetFeatureInfo requests when selecting a record after a search. The point for the GFI request is taken from the selected record's geometry.
```

Comment:
remove the following:

**_The point for the GFI request is taken from the selected record's geometry._**

you have specified it later